unitTests/ui/SmartSnippetSuggestionsTest.ts

```diff
@@ -298,13 +296,44 @@ export function SmartSnippetSuggestionsTest() {
             });
           });
 
-          it('renders the expected shadow content', () => {
-            findClass(ClassNames.RAW_CONTENT_CLASSNAME, true).forEach((content, i) =>
-              expect(content.innerHTML).toEqual(mockSnippet(i).innerHTML)
-            );
+          it('should wrap the snippet in a container in a shadow DOM', () => {
+            getShadowRoots().forEach((shadowRoot, i) => {
+              const [shadowContainer] = expectChildren(shadowRoot, [ClassNames.RAW_CONTENT_CLASSNAME, null]);
+
+              expect(shadowContainer.innerHTML).toEqual(mockSnippet(i).innerHTML);
+            });
+          });
+
+          it('should render the default style', () => {
+            getShadowRoots().forEach(shadowRoot => {
+              const [, styleElement] = expectChildren(shadowRoot, [ClassNames.RAW_CONTENT_CLASSNAME, null]);
+
+              expect(styleElement.innerHTML).toEqual(getDefaultSnippetStyle(ClassNames.RAW_CONTENT_CLASSNAME));
+            });
           });
         });
       });
     });
+
+    describe('with no styling, without sources', () => {
+      beforeEach(async done => {
+        instantiateSmartSnippetSuggestions('');
+        document.body.appendChild(test.env.root);
+        await triggerQuestionAnswerQuery(false);
+        done();
+      });
+
+      afterEach(() => {
+        test.env.root.remove();
+      });
+
+      it('should render the no style', () => {
```

Comment:
Hmm, I do not understand this test description,

Reply:
Woops, made a typo. I meant to write "should render no style", as in an empty style. I'll fix it.